common/travis/manual-ci.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+echo "Checking links"
+RUST_LOG=linkcheck=debug mdbook build manual
+LINKCHECK=$?
+
+# Format them
+printf "Formatting tree"
+vmdfmt -l -w Manual.md
```

Comment:
This isn't actually working correctly:

> error: Unsupported node type HTMLSpan ignored

Folks have expressed dislike towards vmdfmt, since it can lead to single line changes moving things around in a whole paragraph. I don't think it's worth implementing it here. Manual.md is actually very far from conforming to it.

Reply:
removed, it's only linkcheck now.